ga4gh/cli.py

```diff
@@ -34,173 +32,6 @@
 AVRO_LONG_MAX = 2**31 - 1
```

Comment:
It's a bit of a wart that we have this at all --- the protocol should be able to state that end=null means everything on the reference. Fixing the protocol would be the preferred way to get rid of this.